### PR TITLE
Dunkin' Donuts

### PR DESCRIPTION
**Pull request by kyeah**
dunkin' donuts™ has a pretty good iced coffee

cc. @jeffreythewang

### CODE REVIEW

**Comment by faiq** [comment]
:+1:

File: data/research.json [hunk 1]
```diff
@@ -592,6 +592,21 @@
...
+                        "address1" : "266 1st Avenue",
+                        "address2" : "",
+                        "city" : "New York",
+                        "state" : "NY",
+                        "zip" : "10009",
+                        "country" : "USA",
+                        "coords" : {
+                            "lat" : "40.732157",
+                            "long" : "-73.981547",
+                            "zoom" : 17,
```
Comment (jeffreythewang) [inline data/research.json]: Don't use trailing commas. The zoom value is fine (tested locally).

Other than that, LGTM.

**Comment by kyeah** [comment]
Removed trailing comma™

**Comment by jeffreythewang** [comment]
Nice, squash it and I'll merge it in

**Pull request merged**